.circleci/config.yml

```diff
@@ -371,6 +371,11 @@ commands:
           paths:
             - artifacts/blob-report
 
+  #create separate job for html reporting for Stage and Prod
+  create-html-report:
+    steps:
+      - store_artifacts:
+          path: packages/functional-tests/playwright-report
```

Comment:
Q: Does this still create+store artifacts if the job fails?

Reply:
Good question. I changed the way we were executing the workflows. Now it requires the tests to be run before storing artifacts and the tests would be run if builds are successful (I checked that in the previous PRs).